cfpb/consumerfinance.gov
# Autocomplete: Remove xhr wrapper utility and use native fetch instead

## Changes

- Autocomplete: Remove xhr wrapper utility and use native fetch instead


## How to test this PR

1. The autocomplete on the search on http://localhost:8000/ask-cfpb/ should work error-free. You should see the requests in the network tab.
